.github/workflows/sonar.yml

```diff
@@ -0,0 +1,127 @@
+name: "SonarCloud"
+
+on:
+  workflow_run:
+    workflows: [ Build ]
```

Comment:
This should match the `name` from `build.yml`, i.e. `"Build and Unit tests"`

Reply:
Oh sorry! I'll fix this!